template-parts/header/breadcrumbs.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Adds header breadcrumbs
+ *
+ * @package shiro
+ */
+
+$defaults = [
+	'breadcrumb_parent'              => is_singular() ? wp_get_post_parent_id( get_the_ID() ) : null,
+	'breadcrumb_custom_parent_link'  => '',
+	'breadcrumb_custom_parent_title' => '',
+];
+$breadcrumb_args = wp_parse_args( $args, $defaults );
+
+$breadcrumb_parent       = $breadcrumb_args['breadcrumb_parent'];
+$breadcrumb_parent_link  = $breadcrumb_args['breadcrumb_custom_parent_link'];
+$breadcrumb_parent_title = $breadcrumb_args['breadcrumb_custom_parent_title'];
+
+if ( empty( $breadcrumb_parent_link ) && ! empty( $breadcrumb_parent ) ) {
+	$breadcrumb_parent_link = get_the_permalink( $breadcrumb_parent );
+}
+
+if ( empty( $breadcrumb_parent_title ) && ! empty( $breadcrumb_parent ) ) {
+	$breadcrumb_parent_title = get_the_title( $breadcrumb_parent );
+}
+
+$post_ancestors = is_page() ? get_post_ancestors( $breadcrumb_parent ) : [];
+?>
+
+<nav aria-label="Breadcrumbs" class="breadcrumbs">
+	<h2 class="screen-reader-text">You are here:</h2>
+	<ul>
+		<li><a href="/">Home</a></li>
```

Comment:
Should this link go to `home_url()` to get a localized home link?

For that matter, I think "Home' should probably be internationalized.